src/structures/Server.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+import express from "express";
+import { sign } from "tweetnacl";
+import { InteractionData } from "../typings";
+import { Client } from "./Client";
+import { text } from "body-parser";
+
+export class ExpressServer {
+    public readonly app = express();
+    public constructor(public readonly client: Client) {}
+
+    public start(): void {
+        this.app.listen(this.client.config.serverPort, () => {
+            this.client.logger.info(`Express server listening to 0.0.0.0:${this.client.config.serverPort}`);
+        });
+        this.app.use(text({ type: "*/*" }));
+        this.app.post("/interactions", (request, response) => {
```

Comment:
Where is / path? just send `{ status: 200, message: "Hello World" }` in JSON format

Reply:
Okay